int32[] index_group
---
int32 res